opentelemetry-appender-tracing/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## vNext
 
+- Add new `trace_correlation` feature that picks up the trace id and span id
+  from the current OpenTelemetry `Context`.
+- Remove the `experimental_use_tracing_span_context` since `tracing-opentelemetry`
+  now supports activating the OpenTelemetry context for the current tracing span.
```

Comment:
lets link to the relevant setting in `tracing-opentelemetry`, even though its on-by-default.
https://github.com/tokio-rs/tracing-opentelemetry/blob/884b00cf438557733bd9cef9456281bea8c4bea1/src/layer.rs#L842 this I guess?